### DISCUSSION TITLE
Stopping an app store app from updating

### DISCUSSION THREAD

**Discussion by apastuszak** [Homebrew/discussions#4565] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.21
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9d0b7361d1a197949dbb47b761405075f83983b7
Last commit: 6 days ago
Core tap JSON: 11 Jun 23:35 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.41.0 => /opt/homebrew/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Putting non-prefixed coreutils in your path can cause GMP builds to fail.
```

### Description of issue

I recently installed the app fsnotes through homebrew.

To support the author I uninstalled the app using `brew remove fsnotes` and bought it in the App Store.

The problem I have now is that when I run `brew update && brew upgrade` it wants to update fsnotes when a new version comes out.

How do I fix this.  I know I can `brew pin fsnotes`, but I feel like this is not the right solution. How do I get homebrew to forget about this package?

**Comment by p-linnane** [comment]
Can you try `brew uninstall --force fsnotes` and see if the issue persists?
